.storybook/components/DesignTokens/Tier3/Colors.jsx

```diff
@@ -0,0 +1,137 @@
+import React, { Component } from 'react';
+import { Section, Grid, GridItem, Heading } from '../../../../src';
+import filterTokens from '../../../util/filterTokens';
+import { TokenSpecimen } from '../../TokenSpecimen/TokenSpecimen';
+
+export class Tier3ButtonColors extends Component {
```

Comment:
Separated this one out because the layout of this page yields a gigantic snapshot, which breaks chromatic :(